### ISSUE TITLE
Set Up In-Flight Requests Prior to Test Marshal Migration

### ISSUE THREAD

**Issue by m-prodan** [bcgov/foi-flow#2991]
Title of ticket: Set Up In-Flight Requests Prior to Test Marshal Migration

#### Description
Need to simulate a production environment, by having a number of requests in various states in our standard test environment. We need to have the requests staged ahead of the migration from Marshal, to ensure the existing in-flight requests will continue to work as expected.

Requests below should include at least 1 personal, 1 general (Not incl. on hold/fee estimate since those are generals specific). All requests should also be synced to AXIS to ensure we replicate production as much as possible. 

#### DOD
- [x] Identify at least 5 requests that have been in the system longer than ~3 months (various states, if possible) - Matt
- [x] Stage at least 2 requests in the Intake in Progress state - Matt
- [x] Stage at least 2 requests in the Open state - Matt
- [x] Stage at least 2 requests in CFR - Matt
- [x] Stage at least 2 requests in Fee Estimate - Matt
- [x] Stage at least 2 requests in On-Hold
- [x] Stage at least 2 requests in Deduplication
- [x] Stage at least 2 requests in Harms Assessment
- [x] Stage at least 2 requests in Records Review
- [x] Stage at least 2 requests in Consultation
- [x] Stage at least 2 requests in Ministry Sign-Off
- [x] Stage at least 2 requests in Response
- [x] Flag about 5 closed requests (different Ministries) to test searches and access on
- [ ] Add synced requests prior to migration - one in fee estimate, one in on-hold, one in CFR, one in records review

**Comment by m-prodan** [comment]
@lmullane @liseandtea 

Ticket for tracking progress on this one. I've got 13 lines on the DOD, so that could be 4 each (plus one for one lucky person) 

I'll go ahead and take the first 5 items, you two can split the last 8?

**Comment by m-prodan** [comment]
Note for requests after on-hold state, we should try and figure out how an outstanding balance might be collected after the merge.

**Comment by m-prodan** [comment]
Also - these requests should all be synced requests from AXIS - as we want to simulate the merge from a production perspective. Will update the ticket

**Comment by m-prodan** [comment]
https://bcgov.sharepoint.com/:x:/t/03521/EYg8iek3MSxKrp8SvGDwFxQBrbFQkQrsUl4AjqgiYB1FlQ?e=G7yLNv - add requests here when they are set up in test

**Comment by m-prodan** [comment]
All staged and flagged - last thing to do will be to add some synced requests ahead of the merge, adding one more DOD before I close ticket

**Comment by m-prodan** [comment]
I have tested the in-flight requests staged, and ran into minimal issues. I have specific instructions to give to users for requests that are already in the on-hold or fee estimate state, as there are some steps users will need to take if they want to minimize issues. For clarity, the issues I found were in requests that were in fee estimate or on hold, and issues would be run into if users do not follow the instructions we provide.
